db.py

```diff
@@ -0,0 +1,45 @@
+from abc import ABC, abstractmethod
+
+import etcd3 
+
+
+class Db(ABC):
+    '''Abstract class for db.'''
+    @abstractmethod
+    def write(self, key=None, value=None): pass
+
+    @abstractmethod
+    def get(self, key): pass
+
+    @abstractmethod
+    def delete(self, key): pass
+        
+
+class EtcdConnect(Db):
+    '''Actions with etcd db.'''
```

Comment:
If I'm not mistaken PEP says that docstrings should be in triple double quotes. please check this